clang/test/Analysis/array-init-loop.cpp

```diff
@@ -330,3 +330,47 @@ void no_crash() {
 }
 
 } // namespace crash
+
+namespace array_subscript_initializer {
```

Comment:
Could you please put these braces on the same line of the namespace/struct declarations? That way its slightly more compact.